### PR TITLE
replaced x.pool with continental zones and a third server to ntp config

### PR DESCRIPTION
**Pull request by datn**
https://github.com/18F/cg-product/issues/309

### CODE REVIEW

File: bosh-deployment.yml [hunk 1]
```diff
@@ -156,7 +156,6 @@ properties:
 
   blobstore:
     provider: s3
-    server_side_encryption: AES256
```
Comment (jmcarp) [inline bosh-deployment.yml]: y no encrypt?

Comment (jmcarp) [inline bosh-deployment.yml]: Sorry, what I'm asking is why this line changed.

Comment (datn) [inline bosh-deployment.yml]: No idea, not intentional. This is, however, consistent with my current error-ridden track record on even the tiniest change to any file. I will examine and fix -- thanks for catching it.

**Comment by sharms** [comment]
Closes #309

**Pull request merged**